RussianFilter/sections/css_extended.txt

```diff
@@ -14,6 +14,8 @@ kino-teatr.ru#?##root div[class^="relap-"]:contains(РЕКОМЕНДУЕТ) div[
 underverse.su,underver.se#?##sidebar1-wrap > h3:contains(Реклама)
 underverse.su,underver.se#?##sidebar1-wrap > h3:contains(Реклама) + div.sb_back
 underverse.su,underver.se#?##sidebar1-wrap > h3:contains(Реклама) + div.sb_back + div.sb_bottom
+! https://github.com/AdguardTeam/AdguardFilters/pull/70744
```

Comment:
Here we use a task link (not pull request one). I will change it.

Reply:
Sorry, was mistake.